src/components/Article.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function Article({ title, date, text }) {
+  return (
+    <>
```

Comment:
It is an article, so `article` tag would be more appropriate here than just fragment.